catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ExportCommand.java

```diff
@@ -214,6 +227,29 @@ protected Object executeWithSubject() throws Exception {
         return null;
     }
 
+    private StringBuilder getUserInputModifiable() throws IOException {
+        int in;
+        StringBuilder builder = new StringBuilder();
+        while ((in = session.getKeyboard()
+                .read()) != '\r') {
```

Comment:
Is `\r` portable? It probably is, but I think `\n` is supposed to represent newline for all OSes. I feel like I read somewhere that `\n` was supposed to change itself depending on the OS hosting the JVM. (I could be wrong).